packages/eslint-config-godaddy/index.js

```diff
@@ -112,6 +112,7 @@ module.exports = {
     'strict': [2, 'global'],
     'use-isnan': 2,
     'valid-jsdoc': [2, { prefer: { return: 'returns' }, requireReturn: false }],
+    'prefer-promise-reject-errors': 2,
```

Comment:
suggest relaxing this new rule by adding `allowEmptyReject: true` to allow `Promise.reject()` to still work, but `Promise.reject("no")` to error

Reply:
actually, after thinking about it a bit more, disregard. the following code would still produce a catch w/ undefined which I would not want. so even `Promise.reject` should always be called with an error object.
```js
async function example() {
  return Promise.reject();
}
(async () => {
  try {
  	const res = await example();
  } catch (e) {
    console.log('err', e); // undefined
  }
})();
```